src/plugins/lwhpolar/README.md

```diff
@@ -0,0 +1,98 @@
+Polar Coordinate Positioning Plugin
+===================================
+
+Preface
+-------
+This plugin replaces the "rel" relative positioning plugin. 
+It includes and enhances its functionality.
+Using both will confuse the positioning, as the first one called replaces 
+relative positions by absolute ones, thus disabling the second one.
```

Comment:
I encountered some problems when limiting the plugin to the polar part, as the rel plugin already took away some prerequisites by adding absolute locations. Therefore I just wanted to make it clear that only one of them should be used. Currently, `weight` would probably not work.

Reply:
We need to research this more then. Is it the inheritance part of rel that is causing the problem? What I mean is, if neither data-x nor data-rel-x is defined, it will compute and create a data-x, right?

We may need to refactor and share that specific part, or think of some other solution. But, if this is the issue, then using a weight < 10 should help, because your plugin will run before rel.